mbtiles/scripts/cli.py

```diff
@@ -285,6 +309,31 @@ def mbtiles(
                 src.crs, "EPSG:4326", src.bounds[::2], src.bounds[1::2]
             )
 
+            # cutlines must be transformed from CRS84 to src pixel/line
+            # coordinates and then formatted as WKT.
+            if cutline is not None:
+                geoms = [shape(f["geometry"]) for f in cutline]
+                union = unary_union(geoms)
+                if union.geom_type not in ("MultiPolygon", "Polygon"):
+                    raise click.ClickException("Unexpected cutline geometry type")
+                west, south, east, north = union.bounds
```

Comment:
This is the first aspect of using a cutline: it restricts the bounds of the output mbtiles database.